### ISSUE TITLE
error code

### ISSUE THREAD

**Issue by mamod03** [mvpsnet/wireguard4vps#4]
Job for wireguard4vps.service failed because the control process exited with error code.

Getting this error ?
is this because I'm using openvz ?

**Comment by mvpsnet** [comment]
Yes, you don't have access to the TUN/TAP device.

The script is designed for full virtualization - KVM.

**Comment by mamod03** [comment]
yes , turned out not working on OVZ , 6 months to answer what a time :D
